docker-compose.yml

```diff
@@ -58,10 +64,38 @@ services:
       - "8025:8025"
     networks:
       - badgr
+  celery_worker:
+    build:
+      context: .
+      dockerfile: .docker/Dockerfile.dev.api
```

Comment:
I think this should not reference the dev dockerfile

Reply:
Nevermind, it turns out that docker-compose.yml is for development, so this is fine. However it is missing from the docker-compose.prod.yml